### DISCUSSION TITLE
Terraform bitbucket URLs with support for "StabilityDays"

### DISCUSSION THREAD

**Discussion by edmeister** [renovatebot/renovate#12565] [Request Help] [answered]
I was trying to figure out a way to have the `StabilityDays` configuration option working with my Terraform modules. Since they are hosted on Bitbucket, I'm trying to implement this feature by adding an extra clause to https://github.com/renovatebot/renovate/blob/main/lib/manager/terraform/modules.ts#L36. 

However, when running this code, I notice the configuration option is picked up and the updates are checked against the value, but in the end, the PR is still created with the versions that are not old enough.

```
DEBUG: Update has not passed stability days (repository=my/test-repo, branch=renovate/all)
       "depName": "my/test-repo",
       "daysElapsed": 0,
       "stabilityDays": 100
```

What am I missing?

https://github.com/edmeister/renovate/commit/66cdaca1281b02e198f44ba3ea3f62381f71aaad

**Comment by rarkins** [comment, marked as answer]
Try prCreation=not-pending

**Comment by edmeister** [comment]
Thanks Rhys, that did the trick.
